### ISSUE TITLE
Navigate to test/spec

### ISSUE THREAD

**Issue by cmer** [noklesta/SublimeRailsNav#2]
It'd be nice if we could list test/spec files, and navigate directly to the corresponding test/spec based on which file is being edited.

**Comment by noklesta** [comment]
You're absolutely right - it should be in there. I'll see what I can do this week-end.

**Comment by noklesta** [comment]
Ok, I have implemented a new command called `list_rails_tests` (_Simple Rails Navigator: List tests/specs_ in the command palette). You probably need to restart ST2 in order to make Package Control install the new version.

The test for a model or controller that is open in the active view is put at the top of the list, and it also works the other way round (from test to model or controller).

If the application contains a `spec` directory, the plugin will search for specs and ignore the `test` directory; otherwise the `test` directory will be used.

I am planning to include other test-related functionality as well (listing of fixtures, putting tests for views/helpers/routes at the top of the list whenever appropriate).

**Comment by cmer** [comment]
thanks!
